### DISCUSSION TITLE
Toyota Receiving a "Not Logged In" Message

### DISCUSSION THREAD

**Discussion by lunar-studio** [widewing/ha-toyota-na#51] [Q&A]
NOTE: Added the following similar post to issues (sorry but I'm new to discussions and issues on github)
### 
_I had the horrible Entune App, updated it to the newest "Toyota" version, and then suddenly Toyota was asking for a switch and bait subscription to their connected services. I didn't want to pay out of principle but if this works, it might make me reconsider especially if I'm able to start the car via a voice integration.

Just saw your integration and have everything running, except when I try to enter my username and password (tried different things) I receive a "Not Logged In" warning message. Am I doing something incorrect or do I need an active subscription in order to actually log in?

Also tried joining your discord channel but I keep receiving an "Invite Invalid" message._

**Comment by vipul128** [comment]
I'm having the same issue.  My credentials are correct as I'm able to log into the Toyota app on my phone, but when I try to log in using the integration I get the same "not logged in" message.  Any help would be appreciated.

**Reply by jimmycunny** [reply]
I'm also getting the "not logged in' message with the correct creds.

**Comment by jimmycunny** [comment]
This error originated from a custom integration.

Logger: root
Source: custom_components/toyota_na/config_flow.py:34
Integration: Toyota (North America) (documentation, issues)
First occurred: 2:41:48 PM (2 occurrences)
Last logged: 2:45:03 PM

{"authId": "_key_", "callbacks": [{"type": "TextOutputCallback", "output": [{"name": "message", "value": "Invalid UI Locale provided"}, {"name": "messageType", "value": "2"}]}]}
{"authId": "_key_", "callbacks": [{"type": "TextOutputCallback", "output": [{"name": "message", "value": "Invalid UI Locale provided"}, {"name": "messageType", "value": "2"}]}]}

**Reply by badFilename** [reply]
I was using some python code for a while then this started popping up preventing my login. They updated the API to require a locale code but so far I haven't been able to figure out what the codes are.
